scripts/charterafrica.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+docker-compose --env-file apps/charterafrica/.env.local up charterafrica --build -d
+
+sleep 5
+
+docker exec mongodb_container /scripts/db-init.sh
+
```

Comment:
1. Do we need this script? May be a Makefile with various commands should work?
2. Do we need to initialise the database **_every time_** we start the app?